File: application/src/main/java/top/emilejones/hhu/application/platform/dto/mission/enums/MissionStatus.java
package top.emilejones.hhu.application.platform.dto.mission.enums;

public enum MissionStatus {
    RUNNING("运行中"),
    ERROR("任务失败"),
    SUCCESS("运行成功"),
    PENDING("等待中");

    private final String comment;

    MissionStatus(String comment) {
        this.comment = comment;
    }

    public String getComment() {
        return comment;
    }
}

File: domain/src/main/kotlin/top/emilejones/hhu/domain/result/MissionStatus.kt
package top.emilejones.hhu.domain.result

/**
 * 任务状态机的枚举定义。
 * @author EmileJones
 */
enum class MissionStatus(val comment: String) {
    CREATED("创建成功"),
    RUNNING("运行中"),
    ERROR("任务失败"),
    SUCCESS("任务成功"),
    PENDING("等待中")
}
